app/views/v1/sessions/create.json.jbuilder

```diff
@@ -0,0 +1,9 @@
+json.session do
+  if @user
```

Comment:
I wrote a comment supporting @gponsu suggestion. But Github ignored it :-S 

I also suggested to avoid this logic doing something like this at `sessions_controller`:
```ruby
  def create
    @user = SessionService.new.authenticate(params[:email], params[:password])
    return render 'shared/404', status: :unauthorized unless @user
  end
```